cartridges/int_adyen/cartridge/js/pages/checkout/adyen-cse.js

```diff
@@ -0,0 +1,188 @@
+			$('#creditCard_number').removeAttr("disabled").val('');
+			$('#creditCard_expiration_month').val('');
+			$('#creditCard_expiration_year').val('');
+			$('#creditCard_expiration_cvn').val('');	
+			// show the save card input and label because it is a new card
+		    $('#dwfrm_billing_paymentMethods_creditCard_saveCard').show();
+		    $('label[for="dwfrm_billing_paymentMethods_creditCard_saveCard"]').show();
+		} else {
+			populateAdyenCreditCardForm(cardUUID);
+		}
+	});
+	
+	/**
+	 * @function
+	 * @description Fills the Credit Card form with the passed data-parameter and clears the former cvn input
+	 * @param {Object} data The Credit Card data (holder, type, masked number, expiration month/year)
+	 */
+	function setAdyenCCFields(data) {
+	    var $creditCard = $('[data-method="CREDIT_CARD"]');
+	    $('#creditCard_owner').val(data.holder).trigger('change').attr('disabled', 'disabled');
+	    $creditCard.find('select[name$="_type"]').val(data.type).trigger('change').attr('disabled', 'disabled');;
+	    $('#creditCard_number').val(data.maskedNumber).trigger('change').attr('disabled', 'disabled');
+	    $('#creditCard_expiration_month').val(data.expirationMonth).trigger('change');
+	    $('#creditCard_expiration_year').val(data.expirationYear).trigger('change');
+	    $('#creditCard_expiration_cvn').val('').trigger('change');
+	    $creditCard.find('[name$="creditCard_selectedCardID"]').val(data.selectedCardID).trigger('change');
+	    // hide the save card input and label because it is a stored card
+	    $('#dwfrm_billing_paymentMethods_creditCard_saveCard').hide();
+	    $('label[for="dwfrm_billing_paymentMethods_creditCard_saveCard"]').hide();
+	}
+
+	/**
+	 * @function
+	 * @description Updates the credit card form with the attributes of a given card
+	 * @param {String} cardID the credit card ID of a given card
+	 */
+	function populateAdyenCreditCardForm(cardID) {
+	    // load card details
+	    var url = util.appendParamToURL(Urls.billingSelectCC, 'creditCardUUID', cardID);
+	    ajax.getJson({
+	        url: url,
+	        callback: function (data) {
+	            if (!data) {
+	                window.alert(Resources.CC_LOAD_ERROR);
+	                return false;
+	            }
+	            setAdyenCCFields(data);
+	        }
+	    });
+	}
+}
+
+/**
+ * @function
+ * @description Initializes Adyen CSE My Account events
+ */
+function initializeAccountEvents() {
+    $('#add-card-submit').on('click', function (e) {
+    		// TODO: fix this to use IDs and we need to change template to not use name attributes
+        e.preventDefault();
+        var $creditCard = $('#CreditCardForm'),
+        		encryptedData = $('#dwfrm_paymentinstruments_creditcards_newcreditcard_encrypteddata'),
+        		key = SitePreferences.ADYEN_CSE_JS_PUBLIC_KEY,
+        		encryptedDataValue,
+        		options = {};
+      
+        var cardData = {
```

Comment:
let's create a function for that to not duplicate with https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/43/files#diff-48efa7dbf895d9ccd7ba0214da888b35R42